pkg/scheme/scheme.go

```diff
@@ -25,15 +25,18 @@ var (
 			Strict: true,
 		},
 	)
+	localSchemeBuilder = runtime.SchemeBuilder{
+		scyllav1.Install,
+		scyllav1alpha1.Install,
+		cqlclientv1alpha1.Install,
+		monitoringv1.Install,
```

Comment:
I think it should be all of them, including kube. I assume the intended to build a superset so is effectively a deep copy + it's hard to draw a line between monitoring being an external API and kube not being included.

Reply:
makes sense, added.